core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -118,7 +119,8 @@ public FileOutStream createFile(AlluxioURI path, CreateFileOptions options)
     URIStatus status;
     try {
       masterClient.createFile(path, options);
-      status = masterClient.getStatus(path);
+      status = masterClient.getStatus(path, GetStatusOptions.defaults().setLoadMetadataType(
+          LoadMetadataType.Never));
```

Comment:
it makes sense for both createFile and createDirectory to do that (and we should probably do it before 1.5)